### PR TITLE
[TEST] add NEG test case for Device

### PR DESCRIPTION
**Pull request by struss**
This commit will add NEG case for `Device` API.

ONE-vscode-DCO-1.0-Signed-off-by: struss <rrstrous@nate.com>

### CODE REVIEW

**Review by llFreetimell — APPROVED** [review]
LGTM

**Review by hyunsik-yoon — DISMISSED** [review]

File: src/Tests/Execute/Device.test.ts [hunk 1]
```diff
@@ -20,8 +20,8 @@ import {DeviceSpec, HostPCSpec, TizenDeviceSpec} from '../../Backend/Spec';
 import {Device} from '../../Execute/Device';
 
 suite('Device', function() {
-  suite('Device', function() {
-    suite('#contructor()', function() {
+  suite('#Device', function() {
+    test('contructor', function() {
```
Comment (llFreetimell) [inline src/Tests/Execute/Device.test.ts:24]: Sorry for bothering but could you use unique test name even they are different suite?
Our tool has a bug :(

Comment (struss) [reply src/Tests/Execute/Device.test.ts:24]: Oh I will fix it.

**Review by llFreetimell — APPROVED** [review]
LGTM, thanks!

**Review by hyunsik-yoon — APPROVED** [review]

**Pull request merged**